src/createServer.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+
+const { service: userService } = require('./services/users');
+const { service: expenseService } = require('./services/expenses');
+const { router: userRouter } = require('./routes/users');
+const { router: expenseRouter } = require('./routes/expenses');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  userService.init();
+  expenseService.init();
+
+  const app = express();
+
+  app.use(cors());
+  app.use('/users', express.json(), userRouter);
+  app.use('/expenses', express.json(), expenseRouter);
```

Comment:
Remember about DRY. You can `use` `express.json()` only once before your routes.
```suggestion
  const app = express();
  
  app.use(express.json());
  app.use(cors());
  app.use('/users', userRouter);
  app.use('/expenses', expenseRouter);
```